woocommerce/woocommerce
# [e2e] Can Edit and Delete Product Review

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce-quality/issues/610

Closes https://github.com/woocommerce/woocommerce-quality/issues/611

The tests focus on editing and deleting product reviews, verifying the visibility of review elements in the admin interface as a merchant.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Code review.
2. Verify if there are any additional steps or assertions that need to be included
3. Run `pnpm test:e2e-pw ./tests/e2e-pw/tests/merchant/product-reviews.spec.js --headed` and make sure that all the tests are passing.
4. CI jobs should be green.

### Changelog entry


<details>

#### Significance


#### Type


#### Message

Add e2e tests to edit and delete product reviews.

#### Comment

</details>
